api/post_test.go

```diff
@@ -630,19 +628,19 @@ func TestSearchPostsFromUser(t *testing.T) {
 	// wait for the join/leave messages to be created for user3 since they're done asynchronously
 	time.Sleep(100 * time.Millisecond)
```

Comment:
You can take out this sleep as well since we don't need to wait for the join/leave messages